packages/react-components/src/DocsComponents/FluentDocsPage.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import {
+  DocsContext,
+  ArgsTable,
+  Title,
+  Subtitle,
+  Description,
+  HeaderMdx,
+  Primary,
+  PRIMARY_STORY,
+  Stories,
+} from '@storybook/addon-docs';
+
+import { Toc, idToHash } from './Toc.stories';
+
+const CONTAINER_WIDTH = '900px';
+const TOC_WIDTH = '300px';
+
+export const FluentDocsPage = () => {
+  const context = React.useContext(DocsContext);
+  const stories = context.storyStore.getStoriesForKind(context.kind);
+  const primaryStory = stories[0];
+
+  // DEBUG
+  // console.log('FluentDocsPage', context);
+  // console.table(stories.map((s: StoreItem) => ({ id: s.id, kind: s.kind, name: s.name, story: s.story })));
+  // console.table(
+  //   Object.values((context as any).argTypes).map(arg => ({
+  //     name: arg.name,
+  //     description: arg.description,
+  //     type: arg.table?.type?.summary ?? '?',
+  //     default: arg.table?.defaultValue?.summary ?? '-',
+  //   })),
+  // );
```

Comment:
we don't make a habit in the repo of keeping commented code, let's not start now 😉

```suggestion
```